cmd/create/oidcprovider/cmd.go

```diff
@@ -169,7 +170,9 @@ func run(cmd *cobra.Command, argv []string) {
 				"Verify that the cluster operator roles exist and are configured correctly.", clusterKey)
 			os.Exit(1)
 		}
-		reporter.Infof("Creating OIDC provider using '%s'", creator.ARN)
+		if !output.HasFlag() || reporter.IsTerminal() {
```

Comment:
Worth noting I'm not actually adding output support to these commands, but since `output` is set globally then it is respected when running from the cluster creation command.